src/components/v2/V2Project/V2FundingCycleSection/FundingCycleDetails.tsx

```diff
@@ -122,7 +126,11 @@ export default function FundingCycleDetails({
       >
         <Descriptions.Item label={<Trans>Distribution limit</Trans>}>
           <span style={{ whiteSpace: 'nowrap' }}>
-            {distributionLimit ? (
+            {distributionLimitIsInfinite ? (
+              <>∞</>
```

Comment:
I think we should just write `Infinite` here for clarity

Reply:
Good call